test/nuxeo-breadcrumb.test.js

```diff
@@ -102,6 +102,139 @@ suite('nuxeo-breadcrumb', () => {
       expect(title.innerText).to.equal('my file');
     });
 
+    test('Should display the document uid when a document is set and has a title and has a UID', async () => {
+      const uid = breadcrumb.shadowRoot.querySelector('.doc-uid');
+      expect(isElementVisible(uid)).to.be.true;
+      expect(uid.innerText).to.equal('(874fghd)');
+    });
+
+    test('Should display trash icon beside the document title when a document is trashed', async () => {
+      const trashDocument = {
+        'entity-type': 'document',
+        contextParameters: {
+          breadcrumb: {
+            entries: [
+              {
+                path: '/default-domain',
+                title: 'Domain',
+                type: 'Domain',
+                uid: '1',
+              },
+              {
+                path: '/default-domain/workspaces',
+                title: 'Workspaces',
+                type: 'WorkspaceRoot',
+                uid: '2',
+              },
+              {
+                path: '/default-domain/workspaces/my workspace',
+                title: 'my workspace',
+                type: 'Workspace',
+                uid: '3',
+              },
+              {
+                path: '/default-domain/workspaces/my workspace/folder 1',
+                title: 'folder 1',
+                type: 'Folder',
+                uid: '4',
+              },
+              {
+                path: '/default-domain/workspaces/my workspace/folder 1/folder 2',
+                title: 'folder 2',
+                type: 'Folder',
+                uid: '5',
+              },
+              {
+                path: '/default-domain/workspaces/my workspace/folder 1/folder 2/folder 3',
+                title: 'folder 3',
+                type: 'Folder',
+                uid: '6',
+              },
+              {
+                path: '/default-domain/workspaces/my workspace/folder 1/folder 2/folder 3/my file',
+                title: 'my file',
+                type: 'File',
+                uid: '7',
+              },
+            ],
+          },
+        },
+        path: '/default-domain/workspaces/my workspace/folder 1/folder 2/folder 3/my file',
+        title: 'my file',
+        type: 'File',
+        uid: '7ertyyy-hdjkdks-874fghd',
+        isTrashed: true,
+      };
+      breadcrumb.set('document', trashDocument);
+      await flush();
+      const trashIcon = breadcrumb.shadowRoot.querySelector('.trash-icon-parent');
+
+      expect(isElementVisible(trashIcon)).to.be.true;
+    });
+
+    test('Should display trash icon beside the document title when a document is not trashed', async () => {
```

Comment:
I think we are missing a "not" here:
```suggestion
    test('Should not display trash icon beside the document title when a document is not trashed', async () => {
```